x-pack/plugin/inference/build.gradle

```diff
@@ -38,6 +38,169 @@ dependencies {
   clusterPlugins project(':x-pack:plugin:inference:qa:test-service-plugin')
 
   api "com.ibm.icu:icu4j:${versions.icu4j}"
+
+  runtimeOnly 'com.google.guava:guava:32.0.1-jre'
+  api 'com.google.code.gson:gson:2.10'
```

Comment:
do those dependencies all need to be `api`? It seems `implementation` would be more appropriate? I wrote about this here https://groups.google.com/a/elastic.co/g/es/c/Do0hGs7ooJI/m/6nHB9r5NAwAJ a while ago

Reply:
Thank you for sharing! Adjusted it and seems to work fine with [Use implementation instead of api command for 3rd party libraries](https://github.com/elastic/elasticsearch/pull/110090/commits/3d38a4a9642443eb95f21682b86124261c56bf6c) :)